packages/amplify-category-geo/package.json

```diff
@@ -38,6 +38,9 @@
     "lodash": "^4.17.21",
     "uuid": "^8.3.2"
   },
+  "devDependencies": {
+    "@aws-sdk/client-location": "^3.303.0"
+  },
```

Comment:
added for convenience, so that IDE provides syntax/errors assistance .

Reply:
I see `@aws-sdk/client-location@3.41.0` in the yarn.lock already, is it possible to avoid the duplication?

Reply:
It's coming transitively via `e2e tests -> aws-amplify (js libs) -> geo`.

```
yarn why @aws-sdk/client-location
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "@aws-sdk/client-location"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@aws-sdk/client-location@3.312.0"
info Reasons this module exists
   - "_project_#@aws-amplify#amplify-category-geo" depends on it
   - Hoisted from "_project_#@aws-amplify#amplify-category-geo#@aws-sdk#client-location"
info Disk size without dependencies: "11.71MB"
info Disk size with unique dependencies: "47.26MB"
info Disk size with transitive dependencies: "61.24MB"
info Number of shared dependencies: 62
=> Found "@aws-amplify/geo#@aws-sdk/client-location@3.41.0"
info This module exists because "_project_#amplify-e2e-tests#aws-amplify#@aws-amplify#geo" depends on it.
✨  Done in 1.46s.
```